build/source/engine/ffile_info.f90

```diff
@@ -142,13 +142,13 @@ subroutine ffile_info(nHRU,err,message)
   ! set nVar attribute
   forcFileInfo(iFile)%nVars = nVar
 
-  ! inquire nhru dimension size
+  ! inquire nhru dimension size (maybe we need to check the HRU indices are all included instead?)
```

Comment:
We check to see that the HRU sequence in the forcing files exactly matches the HRU sequence in the local attributes file when we read forcing data in (see read_force.f90, starting line 302.